database/schema.sql

```diff
@@ -67,13 +68,14 @@ CREATE TABLE performance_archive(
 
 CREATE TABLE cancel_code(
     cancel_code_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
-    cancel_code CHAR(2) NOT NULL UNIQUE,
-    cause TEXT NOT NULL UNIQUE
+    cancel_code CHAR(2) NOT NULL,
+    cause TEXT NOT NULL,
+    UNIQUE (cancel_code, cause)
```

Comment:
Should both be unique together? I was thinking just the cancel code needs to be unique

Reply:
No that makes sense I've made the changes to the file and to the database